src/main/java/io/fabric8/elasticsearch/plugin/OpenshiftRequestContextFactory.java

```diff
@@ -64,6 +67,9 @@ public OpenshiftRequestContextFactory(final Settings settings, final RequestUtil
                 ConfigurationSettings.DEFAULT_OPENSHIFT_OPS_PROJECTS);
         this.kibanaPrefix = settings.get(ConfigurationSettings.KIBANA_CONFIG_INDEX_NAME, ConfigurationSettings.DEFAULT_USER_PROFILE_PREFIX);
         this.kibanaIndexMode = settings.get(ConfigurationSettings.OPENSHIFT_KIBANA_INDEX_MODE, UNIQUE);
+        this.openshiftMasterUrl = settings.get(ConfigurationSettings.OPENSHIFT_MASTER, ConfigurationSettings.DEFAULT_MASTER);
+        this.openshiftCaPath = settings.get(ConfigurationSettings.OPENSHIFT_CA_PATH, null);
```

Comment:
What is the default CA path?

Reply:
There is no default CA path. If this parameter is not set by the user then the CA detected or set by the K8S plugin is not overwritten.